Signal/src/ViewControllers/PhotoLibrary/PhotoCollectionPickerController.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+import Photos
+import PromiseKit
+
+protocol PhotoCollectionPickerDelegate: class {
+    func photoCollectionPicker(_ photoCollectionPicker: PhotoCollectionPickerController, didPickCollection collection: PhotoCollection)
+}
+
+class PhotoCollectionPickerController: OWSTableViewController, PhotoLibraryDelegate {
+
+    private weak var collectionDelegate: PhotoCollectionPickerDelegate?
+
+    private let library: PhotoLibrary
+    private let lastPhotoCollection: PhotoCollection
```

Comment:
TIOLI: I try to avoid `last` since it can mean final or previous. I think we want `previous` here.

Reply:
Done.